cypress/support/fragments/exportManager/exportManagerSearchPane.js

```diff
@@ -0,0 +1,107 @@
+		cy.wait(1000);
+	},
+	resetJobType() {
+		cy.do(Accordion({ id: 'type' }).find(Button({ icon: 'times-circle-solid' })).click());
+		//Cypress clicks before the UI loads, there is no way to attach waiter to element
+		cy.wait(1000);
+	},
+	searchByScheduled() {
+		//Cypress clicks before the UI loads, there is no way to attach waiter to element
+		cy.wait(1000);
+		cy.do(Accordion({ id: 'status' }).find(Checkbox({ id: 'clickable-filter-status-scheduled' })).click());
+	},
+	searchByInProgress() {
+		//Cypress clicks before the UI loads, there is no way to attach waiter to element
+		cy.wait(1000);
+		cy.do(Accordion({ id: 'status' }).find(Checkbox({ id: 'clickable-filter-status-in-progress' })).click());
+	},
+	searchBySuccessful() {
+		//Cypress clicks before the UI loads, there is no way to attach waiter to element
+		cy.wait(1000);
+		cy.do(Accordion({ id: 'status' }).find(Checkbox({ id: 'clickable-filter-status-successful' })).click());
+	},
+	searchByFailed() {
+		//Cypress clicks before the UI loads, there is no way to attach waiter to element
+		cy.wait(1000);
+		cy.do(Accordion({ id: 'status' }).find(Checkbox({ id: 'clickable-filter-status-failed' })).click());
+	},
+	verifyResult(content) {
+		cy.expect(MultiColumnListCell(including(content)).exists());
+	},
+	searchByBulkEdit() {
+		//Cypress clicks before the UI loads, there is no way to attach waiter to element
+		cy.wait(1000);
+		cy.do(Accordion({ id: 'type' }).find(Checkbox({ id: 'clickable-filter-type-bulk-edit' })).click());
+	},
+	searchByCirculationLog() {
+		//Cypress clicks before the UI loads, there is no way to attach waiter to element
+		cy.wait(1000);
+		cy.do(Accordion({ id: 'type' }).find(Checkbox({ id: 'clickable-filter-type-circulation-log' })).click());
+	},
+	enterStartTime(fromDate, toDate) {
+		cy.do([
+			Accordion({ id: 'startTime' }).clickHeader(),
+			Accordion({ id: 'startTime' }).find(TextField({ name: 'startDate' })).fillIn(fromDate),
+			Accordion({ id: 'startTime' }).find(TextField({ name: 'endDate' })).fillIn(toDate),
+			Accordion({ id: 'startTime' }).find(Button('Apply')).click(),
+		]);
+	},
+	resetStartTime() {
+		cy.do(Accordion({ id: 'startTime' }).find(Button({ ariaLabel: 'Clear selected filters for "[object Object]"' })).click());
+	},
+	enterEndTime(fromDate, toDate) {
+		cy.do([
+			Accordion({ id: 'endTime' }).clickHeader(),
+			Accordion({ id: 'endTime' }).find(TextField({ name: 'startDate' })).fillIn(fromDate),
+			Accordion({ id: 'endTime' }).find(TextField({ name: 'endDate' })).fillIn(toDate),
+			Accordion({ id: 'endTime' }).find(Button('Apply')).click(),
+		]);
+	},
+	resetEndTime() {
+		cy.do(Accordion({ id: 'endTime' }).find(Button({ ariaLabel: 'Clear selected filters for "[object Object]"' })).click());
+	},
+	searchBySystemNo() {
+		cy.do([
+			Accordion({ id: 'isSystemSource' }).clickHeader(),
+			Accordion({ id: 'isSystemSource' }).find(Checkbox({ label: 'No' })).click(),
```

Comment:
Accordion({ id: 'isSystemSource' }) to const too

Reply:
done